README.md

```diff
@@ -481,3 +481,73 @@ If responisve image aspect ratio is added, additional styles inline are generate
 ### Wishlist unlocker -  snowdog/module-wishlist-unlocker
 * This extension allow to show more than 3 items in sidebar wishlist (M2 default is 3)
 * Limit value can be set in admin: Store -> COnfiguration -> Customer -> Wish List -> General option -> Items Limit
+
+## Magepack
+for Magento version < 2.3.6,
+
+### Generate magepack config
+To start using magepack we need to generate magepack config.
+Before start:
+* clear Magento cache
+* compile assest for production mode (in `/tools` directory):
+```
+yarn styles --prod && yarn babel --prod && yarn svg
+```
+* generate config with command (with appropriate path for each site):
+```
+yarn magepackGenerate --cms-url="https://baseUrl/" --category-url="https://baseUrl/categoryPage" --product-url="https://baseUrl/productPage"
+```
+Magepack config will be generated in `/tools` as `magepack.config.js` (which is a symlink to vendor/snowdog/frontools/magepack.config.js).
+You can move this file to main repo or to other location, add this to `.gitignore` and commit changes.
+
+With commited magepack config, during deployment, after assets compilation, run magepack bundling:
+```
+yarn magepackBundle --config <config_path>
+```
+
+### Add assets
+* If you added fonts or external assest that can be load with `preload`, add them in:
```

Comment:
There is a typo `assest`, should be `assets`

Reply:
👍